cypress/integration/utils/index.ts

```diff
@@ -175,12 +178,15 @@ function getAssertionTest(
     cy.get(selectorStr).eq(selectOption!).click();
   }
   if (sqlQuery) {
-    cy.get("[data-cy=sql-editor-collapsed]").click();
     cy.get(selectorStr).get("textarea").clear().type(sqlQuery);
   }
   if (targetPage) {
     cy.visitPage(targetPage, false);
   }
+  if (gridValue) {
+    cy.get('[aria-rowindex="2"]>div').eq(1).type("pk");
+    cy.get('[aria-rowindex="2"]>div').eq(2).type(gridValue);
+  }
```

Comment:
Is there a way to do this without creating a new expectation and using `newExpectationWithTypeString`?

Reply:
For this also I think we should figure out a way to change multiple cells at a time. Could `gridValue` be an array of strings and we can do something like:
```suggestion
  if (gridValues) {
    gridValues.forEach((val, idx) => {
      cy.get('[aria-rowindex="2"]>div').eq(idx+1).type(val);
    })
  }
```

Reply:
just not sure how to do eq() as selectorStr passed to newExpectationWithTypeString, otherwise it will be much better if we could use it!

Reply:
got rid of the newExpectationWithSpreadSheetEditor, used newExpectationWithTypeString.
added gridvalues as input for newExpectationWithTypeString